src/Utils/getUser.ts

```diff
@@ -0,0 +1,22 @@
+export function getUser(user: User): User {
```

Comment:
Is this function never called for unauthenticated users? Because I would expect it to return `null` in such a case.